web/components/requests/FcStudyRequestBulkDetails.vue

```diff
@@ -135,23 +68,36 @@ export default {
         studyRequests[i].dueDate = dueDate;
       }
     },
+    'internalValue.estimatedDeliveryDate': function watchDueDate() {
```

Comment:
this is a lot of copy-paste.  can we abstract these into a single function?  (we could even do a `mapXXX`-style idiom similar to `vuex`...)